spec/requests/user/user_settings_spec.rb

```diff
@@ -238,20 +238,29 @@
       expect(user.reload.profile_updated_at).to be > 2.minutes.ago
     end
 
-    it "disables reaction notifications" do
-      put "/users/#{user.id}", params: { user: { tab: "notifications", reaction_notifications: 0 } }
-      expect(user.reload.reaction_notifications).to be(false)
+    it "disables reaction notifications (in both users and notification_settings tables)" do
```

Comment:
These are the specs that I added to the `users_setting_spec` file to test the `users_setting` and `users_notification_setting` values being set. Are these sufficient? Please advise.